docs/OnlineTutorial/Lemmas.md

```diff
@@ -440,18 +434,18 @@ is not the empty sequence, then when we employ our trick of following the defini
 ``` {.editonly}
 function count(a: seq<bool>): nat
 {
-   if |a| == 0 then 0 else
-   (if a[0] then 1 else 0) + count(a[1..])
+  if |a| == 0 then 0 else
+    (if a[0] then 1 else 0) + count(a[1..])
```

Comment:
```suggestion
  if |a| == 0 then 0 else
  (if a[0] then 1 else 0) + count(a[1..])
```

Reply:
Are you sure about this? The second line is not a new if statement, but the value for the else. I would indent it.